Assets/MixedRealityToolkit-SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -190,10 +198,37 @@ protected virtual void AttachToNewTrackedObject()
                 case TrackedObjectType.Head:
                     TrackTransform(CameraCache.Main.transform);
                     break;
-                // Other cases will come online as ControllerFinder is ported appropriately.
+                case TrackedObjectType.LeftController:
+                    var leftController = GetVisualizerForHand(Handedness.Left);
+                    if (leftController != null)
+                    {
+                        TrackTransform(leftController.gameObject.transform);
+                    }
+                    break;
+                case TrackedObjectType.RightController:
+                    var rightController = GetVisualizerForHand(Handedness.Right);
+                    if (rightController != null)
+                    {
+                        TrackTransform(rightController.gameObject.transform);
+                    }
+                    break;
+
+                    // Other cases will come online as ControllerFinder is ported appropriately.
             }
         }
 
+        private MixedRealityControllerVisualizer GetVisualizerForHand(Handedness hand)
+        {
+            foreach (MixedRealityControllerVisualizer controller in MixedRealityManager.Instance.MixedRealitySceneObjects)
```

Comment:
for?

Reply:
I thought about it.  However, this call is used so infrequently I didn't think we needed to optimise it.
Open to review of course :D (as per standards)

Reply:
We should probably be checking if the controller is null as well, just in case. changing to a for loop might make that a bit simpler.